generators/azure-spring-cloud/index.js

```diff
@@ -417,19 +417,22 @@ for more detailed information.`
 
                 const done = this.async();
                 this.log(chalk.bold('\nDeploying application...'));
-
+                let buildDir = 'target';
```

Comment:
there should be some method or constant to improve this part, but should be ok for now